environment.rb

```diff
@@ -0,0 +1,101 @@
+module Environment
+	class Depot
+		attr_accessor :packs
+		def initialize
+			@packs = {
+				simple_transportation_pack: {
+					intelligence: [:cellphone],
+					items: [:medipack, :chevy_versa]						
+				},		
+				standard_transportation_pack: {
+					intelligence: [:cellphone, :antenna],
+					arsenal: [:colt_1911],
+					items: [:handcuffs, :medipack, :chemistry, :chevy_versa]
+				},
+				  simple_mission_pack: {
+					intelligence: [:infopack, :laptop, :cellphone, :antenna],
+					arsenal: [[:colt_1911], [:handcuffs, :medipack, :chemistry, :financial]]
+				},
+				standar_mission_pack: {
+					intelligence: [:infopack, :laptop, :cellphone, :antenna],
+					arsenal: [:remington_870, :colt_1911, :machete, :hatchet],
+					items: [:handcuffs, :medipack, :chemistry, :financial]
+				}
+			}
+		end
+    end
+
+    class Control
+		attr_accessor :missions
+		@@modes = [:paused, :aborted, :accomplished, :failed]
+		def initialize
+			@missions = {}
+		end
+
+		def new_mission name:, objective:, pack:
+			@missions[name] = {
+				objective: objective,
+				status: :active,
+				pack: pack
+			}
```

Comment:
I would state hashes like this in one line if possible.
`@missions[name] = { objective: objective, status: :active, pack: pack }`